### PR TITLE
feat: add Bedrock Llama2 13b pipeline

### PR DESCRIPTION
**Pull request by tstadel**
First working version of a Llama2-13b pipeline. Let's get this into dC asap :rocket:

### CODE REVIEW

File: pipelines/GenerativeQuestionAnswering_llama2_13b.yaml [hunk 1]
```diff
@@ -0,0 +1,116 @@
...
+    params:
+      model_name_or_path: intfloat/simlm-msmarco-reranker # Fast model optimized for reranking
+      top_k: 4 # The number of results to return
+      batch_size: 20  # Try to keep this number equal or larger to the sum of the top_k of the two retrievers so all docs are processed at once
+  - name: qa_template
+    type: PromptTemplate
+    params:
+      output_parser:
+        type: AnswerParser
+      prompt: >-
```
Comment (sjrl) [inline pipelines/GenerativeQuestionAnswering_llama2_13b.yaml:44]: Is the `-` needed here? Apologies if it's some yaml formatting thing I'm not familiar with, but I thought the `>` is enough?

**Review by sjrl — APPROVED** [review]
Looks good! Excited to try this out

File: pipelines/GenerativeQuestionAnswering_llama2_13b.yaml (hunk 1 above)
Comment (tstadel) [reply pipelines/GenerativeQuestionAnswering_llama2_13b.yaml:44]: Yes, it has a reason. Without it the prompt string would end with a line feed. The llama2 example prompt in notion doesn't have a trailing line feed either. I only tried it this way.

Comment (sjrl) [reply pipelines/GenerativeQuestionAnswering_llama2_13b.yaml:44]: Thanks for the explanation!

**Pull request merged**